src/box/lua/load_cfg.lua

```diff
@@ -377,6 +383,8 @@ local template_cfg = {
     net_msg_max           = 'number',
     sql_cache_size        = 'number',
     txn_timeout           = 'number',
+
+    metrics = 'table',
```

Comment:
Using hierarchic cfg instead of flattened one (like in log, flightrec opts) was proposed here: https://github.com/tarantool/tarantool/issues/7725#issuecomment-1410045326